launcher/ui/pages/global/LauncherPage.cpp

```diff
@@ -446,7 +446,10 @@ void LauncherPage::loadSettings()
         ui->themeComboBoxCat->addItem("BackgroundCat");
         ui->themeComboBoxCat->addItem("Jinx");
         ui->themeComboBoxCat->addItem("Floppa");
-        if(currentCatStyle == "Floppa")
```

Comment:
oioioiioioiooiioio don't touch floppa!

Reply:
FAKE

Reply:
Line 452!!!!! Floppa ain't dying son

Reply:
I can't believe you've done this.

Reply:
Do you want floppa to be the last index then? 🤨

Reply:
This is just a joke review lol

Reply:
We are all clowns in the same circus

Reply:
We are all clowns in the same circus